#
# Replace bcrypt by bcryptjs to avoid having to compile binaries.
#
--- a/backend/package.json
+++ b/backend/package.json
@@ -17,7 +17,7 @@
 		"ajv": "^8.20.0",
 		"archiver": "^8.0.0",
 		"batchflow": "^0.4.0",
-		"bcrypt": "^6.0.0",
+		"bcryptjs": "^3.0.3",
 		"better-sqlite3": "^13.0.3",
 		"body-parser": "^2.3.0",
 		"chalk": "5.6.2",
--- a/backend/models/auth.js
+++ b/backend/models/auth.js
@@ -1,7 +1,7 @@
 // Objection Docs:
 // http://vincit.github.io/objection.js/
 
-import bcrypt from "bcrypt";
+import bcrypt from "bcryptjs";
 import { Model } from "objection";
 import db from "../db.js";
 import { convertBoolFieldsToInt, convertIntFieldsToBool } from "../lib/helpers.js";
--- a/backend/internal/2fa.js
+++ b/backend/internal/2fa.js
@@ -1,5 +1,5 @@
 import crypto from "node:crypto";
-import bcrypt from "bcrypt";
+import bcrypt from "bcryptjs";
 import { createGuardrails, generateSecret, generateURI, verify } from "otplib";
 import errs from "../lib/error.js";
 import authModel from "../models/auth.js";
